### PR TITLE
3.18 RC1 Changelog & Version Bump

### PR DESCRIPTION
**Pull request by jturel**

### CODE REVIEW

**Comment by jturel** [comment]
@chris1984 ready for review :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+### Hosts
+ * Registration fails if Katello is involved and the host is already subscribed, just not registered to this Satellite ([#31050](https://projects.theforeman.org/issues/31050), [6f671bb7](https://github.com/Katello/katello.git/commit/6f671bb7871b1cb40db9b7eef2a8595d11825e07))
+ * reporter_id needs to be set for hypervisor update ([#31004](https://projects.theforeman.org/issues/31004), [dd6bd935](https://github.com/Katello/katello.git/commit/dd6bd93533efebb03ad5a5b8defc5d23086415c4))
+ * Hypervisors upload fails with duplicate UUIDs ([#30826](https://projects.theforeman.org/issues/30826), [59210c07](https://github.com/Katello/katello.git/commit/59210c0704e6935e045d6cb20584618862bac368))
+ * Remove debian packages through content host page ([#30811](https://projects.theforeman.org/issues/30811), [cdbb0049](https://github.com/Katello/katello.git/commit/cdbb0049115a917989bcfb9f213753356c717a11))
+ * Toggling Simple Content Access does not update host Subscription status ([#30758](https://projects.theforeman.org/issues/30758), [75864d0b](https://github.com/Katello/katello.git/commit/75864d0b7bfbb600baebad26b7293f6610a4a119))
+
+### Inter Server Sync
```
Comment (chris1984) [inline CHANGELOG.md:48]: Should we move this guy to `Content Views` where the other export item is?

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+ * Global Registration - Search for @host by :uuid or :hostname ([#30904](https://projects.theforeman.org/issues/30904), [ad6a02a5](https://github.com/Katello/katello.git/commit/ad6a02a502d2300e295ecc134b5c991aaf7f8338))
+ * Extend Global registration endpoint with :activation_key parameter. ([#30676](https://projects.theforeman.org/issues/30676), [0fc2c0a9](https://github.com/Katello/katello.git/commit/0fc2c0a9572b3fa8cc4919e2248a378e4843e100))
+
+### Other
+ * Allow to set prefix for Katello::KatelloUrlsHelper::subscription_manager_configuration_url ([#30819](https://projects.theforeman.org/issues/30819), [2b654767](https://github.com/Katello/katello.git/commit/2b6547675404c4243c4b2d001b71e3ae1659bfa1))
+ * Add pulp3 debian support ([#29421](https://projects.theforeman.org/issues/29421), [cbd77ad8](https://github.com/Katello/katello.git/commit/cbd77ad888cf139f46a65f462634c060f9e6c369))
```
Comment (chris1984) [inline CHANGELOG.md:20]: Should this go under `### Repositories` or something related to content?

**Review by chris1984 — APPROVED** [review]
APJ, left just 2 comments around the changelog

**Comment by jturel** [comment]
@chris1984 updated!

**Review by chris1984 — APPROVED** [review]
LGTM

**Comment by jturel** [comment]
[test katello]

**Comment by jturel** [comment]
[test katello]

**Comment by jturel** [comment]
[test katello]

**Comment by jturel** [comment]
[test katello]

**Comment by jturel** [comment]
[test katello]

**Comment by jturel** [comment]
[test katello]

**Comment by jturel** [comment]
[test katello]

**Comment by jturel** [comment]
[test katello]

**Comment by jturel** [comment]
[test katello]

**Comment by jturel** [comment]
The last build completed to the point of workspace cleanup. With that in mind I'm merging this in

**Pull request merged**